src/fr/poseidonscans/src/eu/kanade/tachiyomi/extension/fr/poseidonscans/PoseidonScans.kt

```diff
@@ -1,14 +1,514 @@
 package eu.kanade.tachiyomi.extension.fr.poseidonscans
 
-import eu.kanade.tachiyomi.multisrc.madara.Madara
+import android.util.Log
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import keiyoushi.utils.parseAs
+import keiyoushi.utils.tryParse
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.jsonArray
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.jsonPrimitive
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrlOrNull
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
 import java.text.SimpleDateFormat
 import java.util.Locale
+import java.util.TimeZone
 
-class PoseidonScans : Madara(
-    "PoseidonScans",
-    "https://poseidonscans.fr",
-    "fr",
-    dateFormat = SimpleDateFormat("dd/mm/yyyy", Locale.FRANCE),
-) {
-    override val useNewChapterEndpoint = true
+class PoseidonScans : HttpSource() {
+
+    override val name = "Poseidon Scans"
+    override val baseUrl = "https://poseidonscans.fr"
+    override val lang = "fr"
+    override val supportsLatest = true
+    override val versionId = 1
```

Comment:
1 is the default value. To bump:
```suggestion
    override val versionId = 2
```